app/src/main/java/nextstep/shoppingcart/MainScreen.kt

```diff
@@ -0,0 +1,47 @@
+package nextstep.shoppingcart
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import nextstep.shoppingcart.component.MainTopBar
+import nextstep.shoppingcart.component.ProductItem
+import nextstep.shoppingcart.model.dummyProducts
+
+@Composable
+fun MainScreen() {
+    Scaffold(
+        topBar = { MainTopBar() }
+    ) { paddingValues ->
+        LazyVerticalGrid(
+            modifier = Modifier
+                .padding(paddingValues = paddingValues)
+                .padding(horizontal = 16.dp),
+            columns = GridCells.Fixed(2),
+            horizontalArrangement = Arrangement.spacedBy(12.dp),
+            verticalArrangement = Arrangement.spacedBy(20.dp)
+        ) {
+            dummyProducts.forEach {
+                item {
+                    ProductItem(
+                        name = it.name,
+                        imageUrl = it.imageUrl,
+                        price = it.price
+                    )
+                }
+            }
```

Comment:
```suggestion
            items(items = dummyProducts) {
                ProductItem(
                    name = it.name,
                    imageUrl = it.imageUrl,
                    price = it.price
                )
            }
```
items 를 사용하시면 forEach를 사용하지 않고 여러 아이템을 넣으실 수 있습니다.